### PR TITLE
fix Issue #55

### PR DESCRIPTION
**Pull request by m-yano**
example message:  

    error" : "missing required parameter. {\"dtstart\":\"2018-04-1T10:14:26Z\",\"dtend\":\"2018-04-1T11:14:26Z\",\"organizer\":\"makoto.yano@jp.fujitsu.com\",\"summary\":\"hoge\",\"description\":\"moge\",\"attendees\":[\"makoto.yano@jp.fujitsu.com\",\"yano.makoto.test@gmail.com\"]}"

### CODE REVIEW

File: bar/90_contents/UpdateEngine/updateVEvent.js [hunk 1]
```diff
@@ -50,6 +50,10 @@ function(request){
     var info = personalBoxAccessor.getString(pathDavName);
     var accInfo = JSON.parse(info);
 
+    if (!(checkParams(request, params))){
+      return createResponse(400, {"error": "missing required parameter. " + JSON.stringify(params)})
```
Comment (dixonsiu) [inline bar/90_contents/UpdateEngine/updateVEvent.js]: You are using params which are from the request. Should you specify which param is missing?

File: bar/90_contents/UpdateEngine/updateVEvent.js [hunk 2]
```diff
@@ -583,3 +587,26 @@ function getAccessToken(bodyData) {
   var res = JSON.parse(response.body);
   return res.access_token;
 }
+
+function checkParams(request, params){
+  try {
```
Comment (dixonsiu) [inline bar/90_contents/UpdateEngine/updateVEvent.js]: Exception does not occur in this method, you don't need to do try/catch I think.

File: bar/90_contents/UpdateEngine/updateVEvent.js (hunk 1 above)
Comment (m-yano) [reply bar/90_contents/UpdateEngine/updateVEvent.js]: Does that mean that the missing parameters are returned in response?

File: bar/90_contents/UpdateEngine/updateVEvent.js (hunk 2 above)
Comment (m-yano) [reply bar/90_contents/UpdateEngine/updateVEvent.js]: I understand.

**Pull request merged**